### PR TITLE
bluefish, new recipe

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: app-editors/bluefish/bluefish-2.2.12.recipe [hunk 1]
```diff
@@ -0,0 +1,162 @@
+SUMMARY="A powerful HTML editor for experienced web designers and programmers"
+DESCRIPTION="Long ProjectX description.
```
Comment (humdingerb) [inline app-editors/bluefish/bluefish-2.2.12.recipe:2]: Oops. :P

File: app-editors/bluefish/bluefish-2.2.12.recipe [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+ - containing
+ - bullet points etc.
+You can also have really long lines that do not fit in 80 characters and use \
+a backslash to wrap the line."
+HOMEPAGE="http://bluefish.openoffice.nl/"
+COPYRIGHT="2014 Developer name
+	2016 Another developer's name"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="http://www.bennewitz.com/bluefish/stable/source/bluefish-$portVersion.tar.gz"
```
Comment (humdingerb) [inline app-editors/bluefish/bluefish-2.2.12.recipe:13]:
```suggestion
SOURCE_URI="https://www.bennewitz.com/bluefish/stable/source/bluefish-$portVersion.tar.gz"
```

Comment (Begasus) [reply app-editors/bluefish/bluefish-2.2.12.recipe:13]: Hence wip label, just wanted to push what I had right now, needs fine-tuning :)

**Comment by humdingerb** [comment]
Sorry, didn't see the wippy. :)

**Comment by Begasus** [comment]
Something strange.

When launched from the Deskbar I see the GTK icons? Also crashes when trying to save something.
BlueFish-GTK

When launched from Terminal I see the Haiku icons, and no crash when saving.
Blue-Fish2

**Comment by threedeyes** [comment]
> Something strange.

Nothing strange, the reason is the absence of XDG environment variables when starting a program from Tracer.
I used a temporary dirty hack by setting these variables manually at application startup.
I'll try to solve this problem better later, but for now I did this dirty hack: https://github.com/haikuports/haikuports/blob/master/app-office/abiword/patches/abiword-3.0.5.patchset#L69

**Comment by Begasus** [comment]
OK, work is as good as done, I've created recipes for unicode_character_database and gucharmap that can be used as a plugin for BlueFish, almost there :)

**Pull request merged**